components/SearchAutocomplete.tsx

```diff
@@ -62,6 +63,7 @@ const SearchAutocomplete: FC<SearchAutocompleteProps> = ({
   const inputRef = useRef<HTMLInputElement>()
   const popoverRef = useRef<HTMLDivElement>()
   const { onOpen, onClose, isOpen } = useDisclosure()
+  const [focusedOption, setFocusedOption] = useState<SearchOption | null>(null)
```

Comment:
Minor (and I'm working on an `eslint` plugin for this), but can we use the `$` prefixes on variables coming from `useState` / hooks generally?

Reply:
Sure, didn't used to use it, but will try to remember. Having eslint rule would be great.